.github/workflows/unit_tests.yml

```diff
@@ -76,10 +76,12 @@ jobs:
         id: compute-test-matrix
         env:
           compute_all_targets: ${{ contains(github.event.pull_request.title, '[RunAllTests]') }}
+          # See: https://docs.github.com/en/webhooks-and-events/webhooks/webhook-events-and-payloads#pull_request.
+          base_commit_hash: ${{ github.event.pull_request.base.sha }}
         # https://unix.stackexchange.com/a/338124 for reference on creating a JSON-friendly
         # comma-separated list of test targets for the matrix.
         run: |
-          bazel run //scripts:compute_affected_tests -- $(pwd) $(pwd)/affected_targets.log origin/develop compute_all_tests=$compute_all_targets
+          bazel run //scripts:compute_affected_tests -- $(pwd) $(pwd)/affected_targets.log $base_commit_hash compute_all_tests=$compute_all_targets
```

Comment:
TODO: document this in PR description & its benefit for chained PRs.

TODO: verify whether this is sufficient for merge queues.

Reply:
Skipping the merge queues bit since that's part of #3926. Description otherwise updated.